documentation/src/main/asciidoc/public/reference/_indexing-listener-triggered.asciidoc.adoc

```diff
@@ -95,7 +95,7 @@ See <<indexing-plan-synchronization,this section>> for more information.
 
 include::../components/_incubating-warning.adoc[]
 
-In some scenarios, it might be helpful to pause the indexing programmatically, for example,
+In some scenarios, it might be helpful to pause the <<architecture-hsearch-indexing,indexing>> programmatically, for example,
```

Comment:
```suggestion
In some scenarios, it might be helpful to pause the <<architecture-hsearch-indexing,explicit and listener-triggered indexing>> programmatically, for example,
```